src/hb-subset-input.cc

```diff
@@ -448,22 +448,30 @@ hb_subset_input_pin_axis_location (hb_subset_input_t  *input,
 #endif
 #endif
 
-#ifdef HB_EXPERIMENTAL_API
 /**
  * hb_subset_preprocess
```

Comment:
```suggestion
 * hb_subset_preprocess:
```

Reply:
You need to also move it out of Private section in `docs/harfbuzz-sections.txt` for the documentation to be generated.